docs/kernels/index.md

```diff
@@ -0,0 +1,25 @@
+# Kernels
+
+JupyterLite Kernels implement [Jupyter Kernel Messaging][jkm] in the browser with the
+help of [`mock-socket`][mock-socket].
+
+| Feature       | Message                                       | [pyolite](./pyolite.md) | [javascript](./js.md) | [p5](./p5.md) |
+| ------------- | --------------------------------------------- | ----------------------- | --------------------- | ------------- |
+|               | Language                                      | Python 3.8              | Browser JS            | JS + p5.js    |
+| Start session | `kernel_info_request`<br/>`kernel_info_reply` | ✔️                      | ✔️                    | ✔️            | ✔️ |
+| Run code      | `execute_request`<br/>`execute_reply`         | ✔️                      | ✔️                    | ✔️            | ✔️ |
+| Completion    | `complete_request`<br/>`complete_reply`       | ✔️                      | ✔️                    | ✔️            |
```

Comment:
Technically we don't really have completion yet for Pyolite, and the one for JS is really basic.